--- origsrc/ggz-txt-client-0.0.14.1/ggz-txt/output.c	2008-01-05 13:02:52.000000000 -0600
+++ src/ggz-txt-client-0.0.14.1/ggz-txt/output.c	2008-06-06 00:06:27.703125000 -0500
@@ -35,6 +35,9 @@
 #ifdef HAVE_SYS_IOCTL_H
 #  include <sys/ioctl.h>
 #endif
+#ifdef __CYGWIN__
+#  include <termios.h>
+#endif
 #include <time.h>
 
 #include <ggzcore.h>
--- origsrc/ggz-txt-client-0.0.14.1/tttxt/main.c	2008-01-05 13:02:51.000000000 -0600
+++ src/ggz-txt-client-0.0.14.1/tttxt/main.c	2008-06-06 00:07:12.890625000 -0500
@@ -18,6 +18,10 @@
  */
 
 #include <sys/ioctl.h>
+#ifdef __CYGWIN__
+#  include <termios.h>
+#  include <sys/time.h>
+#endif
 #include <stdio.h>
 #include <stdlib.h>
 #include <unistd.h>
